docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -237,6 +234,25 @@ def get_data_loaders(batch_size: int, model):
     return dataloader_train_ssl, dataloader_train_kNN, dataloader_test
 
 
+class CosineWarmupScheduler(torch.optim.lr_scheduler.LambdaLR):
```

Comment:
As we use this class in many places it would maybe be better to move it into its own file. I would propose `lightly/utils/scheduler.py`. Could you also move the `cosine_schedule` method from `lightly/models/utils.py` there and update all the imports?